docs/conf.py

```diff
@@ -58,65 +55,30 @@
 
 autosummary_generate = True
 
-# templates_path = ['_templates']
-
-# The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-#
 source_suffix = ['.rst', '.md']
 
-# The master toctree document.
-master_doc = 'index'
-
-# General information about the project.
-project = u'Ocean Documentation'
-copyright = u'D-Wave Systems Inc'
-author = u'D-Wave Systems Inc'
-
-# The version info for the project you're documenting, acts as replacement for
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-#
-# The short X.Y version.
-import dwaveoceansdk.package_info as sdk
-
-version = sdk.__version__
-# The full version, including alpha/beta/rc tags.
-release = version
-
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-#
-# This is also used if you do content translation via gettext catalogs.
-# Usually you set "language" from the command line for these cases.
-language = None
-
+root_doc = 'index'  # before Sphinx 4.0, named master_doc
 
 add_module_names = False
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This patterns also effect to html_static_path and html_extra_path
+
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
-linkcheck_ignore=[r'.clang-format',                   # would need symlink
-                  r'setup.cfg',                       # would need symlink (for dimod)
-                  r'https://cloud.dwavesys.com/leap', # redirects, many checks
-                  r'https://scipy.org',               # ignores robots
-                  r'LICENSE',                         # would need symlink, checked by submodule
-                  r'CONTRIBUTING',                    # would need symlink, checked by submodule
-                  ]
+linkcheck_ignore = [r'.clang-format',                    # would need symlink
+                    r'setup.cfg',                        # would need symlink (for dimod)
+                    r'https://cloud.dwavesys.com/leap',  # redirects, many checks
+                    r'https://scipy.org',                # ignores robots
+                    r'LICENSE',                          # would need symlink, checked by submodule
+                    r'CONTRIBUTING',                     # would need symlink, checked by submodule
+                    ]
 
-# The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
-# If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
 
 doctest_global_setup = """
-from __future__ import print_function, division
 import operator         # Used by dwave-binarycsp
```

Comment:
No longer needed (dwave-binarycsp examples removed)

Reply:
looks like we're still using it somewhere, so will restore it for now